### DISCUSSION TITLE
Pin Action Bar When Reading Email

### DISCUSSION THREAD

**Discussion by Zadok33** [tutao/tutanota#5274] [Ideas]
I use the Tutanota app on Windows. When I read an email I often need to scroll to read its entirety. When I get to the end I decide if I want to delete it, reply, file it, forward, etc. Each time I finish reading an email I need to scroll all the way to the top to click the desired button on the action bar.

I would appreciate the option of pining the action bar so it remains on top as I scroll through any email. This way I when I finish reading my email I can choose what I want done with it without needing to scroll to the top.

**Comment by serrq** [comment]
Just you need of "back above" button in the bottom of the email.

**Reply by Zadok33** [reply]
Yes, that will work as well.
